bundles/org.openhab.binding.shelly/README.md

```diff
@@ -0,0 +1,453 @@
+
+
+
+### Other devices
+
+The thing definiton fo the following devices is primarily. If you have one of those devices send a PM to marks7017 and we could work on the implementation/testing.
+
+- thing-type: shellysmoke
+- thing-type: shellysense
+- thing-type: shellyplug
+
+
+## Full Example
+
+Note: PaperUI is recommended, if you want to use text files make sure to replace the thing id from you channel definition 
+
+* .things
+
+/* Shelly 2.5 Roller */
+Thing shelly:shelly25-roller:XXXXX1 "Shelly 25 Roller XXXXX1" @ "Home Theater" [deviceIp="x.x.x.x", userId="", password=""]
+Thing shelly:shelly25-roller:XXXXX2 "Shelly 25 Roller XXXXX2" @ "Living Room" [deviceIp="x.x.x.x", userId="admin", password="secret"]
+
+
+/* Shelly 2.5 Relays */
+Thing shelly:shelly25-relay:XXXXX3 "Shelly 25 Relay XXXXX3" @ "Hall Way" [deviceIp="x.x.x.x", userId="", password=""]
+Thing shelly:shelly25-relay:XXXXX4 "Shelly 25 Relay XXXXX4" @ "Dining Room" [deviceIp="x.x.x.x", userId="", password=""]
+Thing shelly:shelly25-relay:XXXXX5 "Shelly 25 Relay XXXXX5" @ "Bed Room" [deviceIp="x.x.x.x", userId="", password=""]
+
+* .items
+
+Switch Shelly_XXXXX3_Relay "Garage Light" (garage) {channel="shelly:shelly1:XXXXX3:relay#output"}
+Switch Shelly_XXXXX3_OverPower "Garage Light Over Power" (garage) {channel="shelly:shelly1:XXXXX3:relay#overpower"}
+Switch Shelly_XXXXX3_OverTemp "Garage Light Over Temperature" (garage) {channel="shelly:shelly1:XXXXX3:relay#overtemperature"}
+Number Shelly_XXXXX3_AutoOnTimer "Garage Light Auto On Timer" (garage) {channel="shelly:shelly1:XXXXX3:relay#autoOn"}
+Number Shelly_XXXXX3_AutoOffTimer "Garage Light Auto Off Timer" (garage){channel="shelly:shelly1:BA2F18:relay#autoOff"}
+Switch Shelly__TimerActive "Garage Light Timer Active" (garage) {channel="shelly:shelly1:BA2F18:relay#timerActive"}
+
+/* Power Meter */
+Number Shelly_BA2F18_Power "Bath Room Light Power" (bathroom) {channel="shelly:shelly1:BA2F18:meter#currentWatts"}
+
+
+* .rules
+
+reading colors from color picker:
+import org.openhab.core.library.types.*
+
+rule "color" 
+when
+    Item ShellyColor changed
+then
+    var HSBType hsbValue = ShellyColor.state as HSBType
+    var int redValue = hsbValue.red.intValue
+    var int greenValue = hsbValue.green.intValue
+    var int blueValue = hsbValue.blue.intValue
+end
+
+
+# Additional Notes
+
+## General
+
+You should use firmware version 1.5.0 or never. It might be that the binding is working with older versions, but this was never tested.
+
+If you gave multiple network interfaces you should check openHAB's default setting.
+Open PaperUI and go to Configuration->System->Network Settings and verify the selected interface. If the Shelly devices are not on the same network you could try to add them manually. However, devices in different networks have not been tested yet (please post a comment in the community thread if you are successful).
+
+## Reporting a problem/bug
```

Comment:
Please remove this block. This is generic information, not specific for a binding and should not be in each binding documentation.